.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,6 +0,0 @@
----
-blank_issues_enabled: true
```

Comment:
I wonder whether this is the default (I personally found the ability to create blank issues useful).

Reply:
If we use it, then we can enable it, for sure. I deleted it because of the security stuff, but it had this setting too. I will add it, no matter if it is the default or not (that way we make it explicit).